content/influxdb/cloud-serverless/reference/cli/influx/org/members/add.md

```diff
@@ -15,7 +15,7 @@ metadata: [influx CLI 2.0.0+, InfluxDB OSS only]
 The `influx org members add` command adds members to organizations in **InfluxDB OSS 2.x**,
 but cannot add organization members to **InfluxDB Cloud Serverless**.
 For information about inviting users into your InfluxDB Cloud Serverless organization,
-see [Manage users](/influxdb/cloud/organizations/users/).
+see [Manage users](/influxdb/cloud/admin/organizations/users/).
```

Comment:
We need to dupe this content for Serverless, don't we - so we're not bouncing them to cloud2?